sympy/simplify/simplify.py

```diff
@@ -580,7 +603,7 @@ def simplify(expr, ratio=1.7, measure=count_ops, rational=False, inverse=False,
     sympy.assumptions.refine.refine : Simplification using assumptions.
     sympy.assumptions.ask.ask : Query for boolean expressions using assumptions.
     """
-
+    expr = simplify_mod_addition(expr)
     def shorter(*choices):
```

Comment:
We would not want to add this simplification call at the beginning of the simplify function.